src/mnemonics/german.h

```diff
@@ -1678,11 +1678,11 @@ namespace Language
         "Zündung",
         "Zweck",
         "Zyklop"
-      }), 4)
-    {
+      }), 4)
+    {

```

Comment:
That looks suspect... looks like a noop, also found in several places, and random #endif changes...